lighthouse-cli/performance-experiment/report/perf-x-report-generator.js

```diff
@@ -20,14 +20,23 @@
 
 const fs = require('fs');
 const path = require('path');
+const Handlebars = require('handlebars');
 const ReportGenerator = require('../../../lighthouse-core/report/report-generator');
+const ConfigPanelFormatter = require('../formatters/config-panel');
 
 class PerfXReportGenerator extends ReportGenerator {
   getReportJS(reportContext) {
     const scriptArr = super.getReportJS(reportContext);
-    scriptArr.push(fs.readFileSync(path.join(__dirname, './injected-script.js'), 'utf8'));
+    scriptArr.push(fs.readFileSync(path.join(__dirname, 'scripts/perf-x.js'), 'utf8'));
     return scriptArr;
   }
+
+  _registerFormatters(audits) {
+    super._registerFormatters(audits);
+    const configPanelTemplate = Handlebars.compile(ConfigPanelFormatter.getFormatter('html'));
```

Comment:
since this is loading the partial differently than how it's done in `_registerFormatters`, would it be better to just load the partial file here instead of making `ConfigPanelFormatter` a `Formatter`?